### ISSUE TITLE
Improving model-training notebook and 'trainers.py'

### ISSUE THREAD

**Issue by pollytur** [sinzlab/sensorium#112]
1. The standard trainer will not stop and will not give any warning if the loss when to 'nan', it will just keep training.
2. It would be nice to add in the example notebooks  `'track_training': True',  'verbose': True` for the `trainer_config` (as its crucially needed for other models experiments)

**Comment by KonstantinWilleke** [comment]
Hi - yes I agree that (1) is an important feature. Has the validation accuracy been a NaN value? 

Same for (2), it'd be a great addition to make it more user-friendly. If you already developed your own tracking, feel free to issue a PR against the current main.

Thanks a lot again for your helpful suggestions.
